### ISSUE TITLE
Android after onPause and onResume()  will get  "get_target_buffer:967: winsysp_window_buffer_get failed 12299" error

### ISSUE THREAD

**Issue by wu1274704958** [SFML/SFML#1489]
after onPause and onResume()  will get  "get_target_buffer:967: winsysp_window_buffer_get failed 12299" error

**Comment by MarioLiebisch** [comment]
Yes, this is a known issue. As a temporary workaround, try to call `sf::RenderWindow::setActive(false)` when you're notified about leaving (`sf::Event::LostFocus` or `sf::Event::MouseLeft`). This will ensure the context is properly recreated/reacquired once you return to the app.
